model/src/main/scala/hmda/model/fi/Filing.scala

```diff
@@ -0,0 +1,9 @@
+package hmda.model.fi
+
+sealed trait FilingStatus
+case object NotStarted extends FilingStatus
+case object InProgress extends FilingStatus
+case object Completed extends FilingStatus
+case object Cancelled extends FilingStatus
+
+case class Filing(id: String = "", fid: String = "", status: FilingStatus = NotStarted)
```

Comment:
Is the `id` the filing period (it seems to be from the institutions api)? If so it might be good to make that explicate in the variable name here and in the filing persistence actor.